sycl/doc/extensions/proposed/sycl_ext_oneapi_current_device.asciidoc

```diff
@@ -0,0 +1,140 @@
+= sycl_ext_oneapi_current_device
+
+
+:source-highlighter: coderay
+:coderay-linenums-mode: table
+
+// This section needs to be after the document title.
+:doctype: book
+:toc2:
+:toc: left
+:encoding: utf-8
+:lang: en
+:dpcpp: pass:[DPC++]
+
+// Set the default source code type in this document to C++,
+// for syntax highlighting purposes.  This is needed because
+// docbook uses c++ and html5 uses cpp.
+:language: {basebackend@docbook:c++:cpp}
+
+
+== Notice
+
+[%hardbreaks]
+Copyright (C) 2024 Intel Corporation.  All rights reserved.
```

Comment:
Just a nit before merging, should it be 2025 instead of 2024?

Reply:
Copyright dates should be the first year of publication.  I think James started drafting this PR in 2024, so that seems like the right date.